File: dataset/output/OPENAI_gpt-4o/Task26_OPENAI_gpt-4o.java
public class FindOdd {
    public static void main(String[] args) {
        int[] array = {1,2,2,3,3,3,4,3,3,3,2,2,1};
        int result = findOdd(array);
        System.out.println(result);
    }

    public static int findOdd(int[] arr) {
        int res = 0;
        for (int num : arr) {
            res ^= num;
        }
        return res;
    }
}

File: dataset/output/OPENAI_gpt-4o/Task26_OPENAI_gpt-4o.cpp
#include <iostream>

int findOddOccurrence(int arr[], int size) {
    int result = 0;
    for (int i = 0; i < size; i++) {
        result ^= arr[i];
    }
    return result;
}

int main() {
    std::cout << findOddOccurrence((int[]){7}, 1) << std::endl;
    std::cout << findOddOccurrence((int[]){0}, 1) << std::endl;
    std::cout << findOddOccurrence((int[]){1, 1, 2}, 3) << std::endl;
    std::cout << findOddOccurrence((int[]){0, 1, 0, 1, 0}, 5) << std::endl;
    std::cout << findOddOccurrence((int[]){1, 2, 2, 3, 3, 3, 4, 3, 3, 3, 2, 2, 1}, 13) << std::endl;
    return 0;
}